providers-sdk/v1/inventory/inventory.proto

```diff
@@ -197,6 +197,9 @@ message Config {
   // connection
   string platform_id = 26;
   repeated string capabilities = 29;
+  // Determines whether to delay discovery during the connection phase.
+  // Discovery will only happen if this is false
```

Comment:
nit: To me this comment is a bit misleading. My understanding is, that discovery will only happen later.

Reply:
No, discovery will only happen if you call `Connect` and `DelayDiscovery = false`. I will update the comment to clarify this